### PR TITLE
added DEX support for BatchNormRelu

### PR DESCRIPTION
**Pull request by pruthviIntel**

### CODE REVIEW

File: src/ngraph/runtime/cpu/builder/batch_norm.cpp [hunk 1]
```diff
@@ -324,7 +325,17 @@ namespace ngraph
...
             }
 
+            template <>
+            void Builder::BUILDER_DECL(ngraph::op::BatchNormRelu)
+            {
+                if (!mkldnn_utils::use_mkldnn_kernel(node))
+                {
+                    throw ngraph_error("BatchNormRelu is only supported with 4-D MKLDNN kernel.");
+                }
+                build_batch_norm(external_function, node, args, out, true);
```
Comment (jbobba) [inline src/ngraph/runtime/cpu/builder/batch_norm.cpp]: you will need to templatize build_batch_norm

Comment (pruthviIntel) [reply src/ngraph/runtime/cpu/builder/batch_norm.cpp]: jayaram, isn't it by flipping relu  in `build_batch_norm ` flag we can control whether to execute Reluas post op, still do we need to tempaltize it?

Comment (jbobba) [reply src/ngraph/runtime/cpu/builder/batch_norm.cpp]: There is a static_cast to op type in build_batch_norm

Comment (pruthviIntel) [reply src/ngraph/runtime/cpu/builder/batch_norm.cpp]: ah thanks for pointing , i overlooked that part :). will fix it.
